Makefile

```diff
@@ -41,7 +41,7 @@ docker-run-cloud:
 #  Meshery Cloud for user authentication.
 run-local-cloud:
 	cd cmd; go clean; rm meshery; go mod tidy; go build -tags draft -a -o meshery; \
-	SAAS_BASE_URL="http://mesherylocal.layer5.io:9876" \
+	SAAS_BASE_URL="http://localhost:9876" \
```

Comment:
Looks good.

FYI - I believe the reason that it was set to mesherylocal is for anyone who might be developing, but running meshery-cloud in docker as opposed to on their host directly.

Reply:
Yes, I will change it back in the final commit for sure. This was it was easier to do so while working in my system.